### PR TITLE
feat(cli): add vscode launch tasks in app

### PR DESCRIPTION
**Pull request by jannyHou**
Signed-off-by: jannyHou <juehou@ca.ibm.com>

connect to https://github.com/strongloop/loopback-next/issues/6463

Added "debug mocha test", "debug mocha test in current file", "debug app" in this PR.
For debugging current test, the mocha-current-file is needed. I am thinking, should we move it to `@loopback/build` to avoid duplicate?

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [x] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/cli/generators/project/templates/.vscode/launch.json
Comment (dhmlau) [inline packages/cli/generators/project/templates/.vscode/launch.json:2]: I think we need to add the launch program for starting the application, i.e. something like this:
```
{
  // Use IntelliSense to learn about possible attributes.
  // Hover to view descriptions of existing attributes.
  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
  "version": "0.2.0",
  "configurations": [
    {
      "type": "node",
      "request": "launch",
      "name": "Launch Program",
      "skipFiles": [
        "<node_internals>/**"
      ],
      "program": "${workspaceFolder}/index.js",
    }
  ]
}

```

File: packages/cli/generators/project/templates/.vscode/launch.json
Comment (dhmlau) [inline packages/cli/generators/project/templates/.vscode/launch.json:21]: we may not want to turn on this by default?

**Review by dhmlau — COMMENTED** [review]
Thanks @jannyHou for adding this. 
I think it might be good to add some description around debugging LB application in the troubleshooting guide, as a separate PR.

**Comment by jannyHou** [comment]
It's really weird, all tests pass on my local.

**Review by dhmlau — APPROVED** [review]
LGTM. Might want to have another approval before landing since it changes the template. Thanks.

File: packages/cli/generators/project/templates/.vscode/launch.json
Comment (raymondfeng) [inline packages/cli/generators/project/templates/.vscode/launch.json:10]: By default, we will have `dist/index.js` instead of root level `index.js` for applications generated by `lb4`.

Comment (jannyHou) [reply packages/cli/generators/project/templates/.vscode/launch.json]: Good catch. I verified with a new app, `${workspaceFolder}/dist/index.js` is correct 👍

**Review by raymondfeng — APPROVED** [review]

**Comment by dhmlau** [comment]
related to https://github.com/strongloop/loopback-next/issues/6613

**Pull request merged**
